contents/blog/why-redesign.mdx

```diff
@@ -0,0 +1,171 @@
+
+Our current UI has served its purpose, but as we grow as a company, the possibilities of what can be achieved in PostHog have grown – as have our ambitions.
+
+We’re launching a reskin this month as the first step in a series of many milestones around the experience of using PostHog. I’d like to share some of the reasoning behind our changes, and the longer-term vision we’re aiming to realize in the future.
+
+## Why we're making changes
+
+Sometimes it feels like designers change things for the sake of change. I promise, this is not one of those times. Here's some of the thinking that's gone into it:
+
+### 1. PostHog isn’t product analytics - it’s a platform
+
+As of this writing, we’re at five products with three more in beta. Interoperability will be a key differentiator for PostHog.
+
+Underlying every query is our new [HogQL querying engine](/docs/hogql). This means it’s now possible to display a graph of users who match a set of criteria, then pivot to watching a playlist of session recordings, then toss those users into a cohort, enable a feature flag, and send them an in-app survey – all stemming from the same underlying query.
+
+This means PostHog needs to be flexible, visually agnostic, and adaptable to pivot to whatever type of querying is being done.
+
+### 2. An experience that sparks joy ✨
+
+We’ve all used web-based products we _love_ and also those that drive us _absolutely insane_. We want to be in the former category. Our goal is to make PostHog feel to product engineers as Figma is for designers, VS Code is for engineers, or Linear is for ~product managers~ everybody.
+
+So what does a better experience look like for product engineers? For starters, dark mode, keyboard shortcuts, better data density, and tools that make it easier for product engineers to do their jobs. But this is just the tip of the iceberg.
+
+(Alongside this release, we’re also launching a preview of [Notebooks](#), a new platform feature that lets you explore data in an unstructured way.)
+
+### 3. Flexibility
+
+Some of our customers use all our products. Some only use one or two. The experience should feel like it was designed specifically for however you use PostHog.
+
+## The biggest changes you'll see today
+
+This initial release of our updated UI is largely focused on aesthetic changes, though it does come with some new features.
+
+But more importantly, it lays the groundwork for future improvements. **Some things may still feel a bit janky, but there are lots of exciting plans. This is a work in progress, and I hope you'll bear with us as we continue to iterate in the coming months.**
+
+Here's some of the design changes (and a few new features) you'll see today:
+
+### 1. Colors, data density, dark mode
+
+_Data density, dark mode, developers._ Now you can see more information on the screen, and it won't make your eyes bleed.
+
+You can change between light, dark, and a mode that will match your system preference under the account menu.
+
```

Comment:
The redesign is also about aligning the PostHog app's design language with that of our brand (website & marketing) – not the most important thing for a user by far, but it's worth mentioning